test/vagrant-service-manager/binary_handlers/adb_docker_binary_handler_test.rb

```diff
@@ -54,6 +54,11 @@ module ServiceManager
       end
 
       it 'should validate download url' do
+        @handler.build_download_url
+        assert_silent { @handler.validate_url }
```

Comment:
> pass the url into the method instead of using @url.

What will be method signature here ? `def validate_url(url)` ? This doesn't seems to be object oriented.

> Why does validate_url not return something true or something?

Thought of that but then we are capturing the return value in `install` method. Whats its purpose of returning?